modules/layers/src/geojson-layer/geojson-layer.js

```diff
@@ -294,48 +382,142 @@ export default class GeoJsonLayer extends CompositeLayer {
         layerProps.lines
       );
 
+    let pointLayerProps;
+    let pointLayerUpdateTriggers;
+    if (pointType === 'circle') {
+      pointLayerProps = {
+        stroked,
+        filled,
+        radiusUnits: sizeUnits,
+        radiusScale: sizeScale,
+        radiusMinPixels: sizeMinPixels,
+        radiusMaxPixels: sizeMaxPixels,
+        lineWidthUnits,
+        lineWidthScale,
+        lineWidthMinPixels,
+        lineWidthMaxPixels,
+
+        getFillColor: this.getSubLayerAccessor(getFillColor),
+        getLineColor: this.getSubLayerAccessor(getLineColor),
+        getRadius: this.getSubLayerAccessor(getRadius),
```

Comment:
Why keep the term `radius` here if everything else is renamed to `size`?

Reply:
I've changed this to getSize for consistency

Reply:
I've changed it to getSize